modules/internal/ChapelComplex_forDocs.chpl

```diff
@@ -38,8 +38,27 @@
 
 /*
 
-Complex is a predefined structured type in Chapel. It is specified
-in the Types chapter of the Chapel Language Specification.
+Complex is a predefined structured type in Chapel.
+
+Like the integral and real types, the complex types can be parameterized
+by the number of bits used to represent them. A complex number is
+composed of two real numbers so the number of bits used to represent a
+complex is twice the number of bits used to represent the real numbers.
+The default complex type, ``complex``, is 128 bits; it consists of two
+64-bit real numbers. The complex types that are supported are
+machine-dependent, but usually include ``complex(64)`` and
+``complex(128)``.
+
+The real and imaginary components can be accessed via the methods ``re``
+and ``im``. The type of these components is real. The standard :mod:`Math`
```

Comment:
Actually, the type is *a* `real` but it might be `real(32)`

Reply:
Could you clarify which part you would want to change?

Reply:
"The type of these components is real"

-> 

"The type of these components is a ``real`` type"